source/common/router/config_impl.cc

```diff
@@ -384,6 +384,8 @@ CorsPolicyImpl::CorsPolicyImpl(const envoy::config::route::v3::CorsPolicy& confi
   if (config.has_allow_credentials()) {
     allow_credentials_ = PROTOBUF_GET_WRAPPED_REQUIRED(config, allow_credentials);
   }
+  allow_private_network_access_ =
+      PROTOBUF_GET_WRAPPED_OR_DEFAULT(config, allow_private_network_access, false);
```

Comment:
The `allow_private_network_access_` always has a value, may we shouldn't use `absl::optional` for it?

Reply:
Sure.